Dynatrace/dynatrace-operator
# Adds csidriver.kubeletPath to values.yaml

# Description
Adds `csidriver.kubeletPath` to values.yaml. 
In some kubernetes variants the path to kubelet is different so the csidriver needs to be configured to work

## How can this be tested?
Run `helm template ...` when `csidriver.kubeletPath` is set  to something and check the output


## Checklist
